### PR TITLE
yes + boop

### PR DESCRIPTION
**Pull request by jbanate**
this all seems about right
- Added a cap to how fast crep can send messages at half a second
- Crep can now parse fractions of a second (!p ccrep 5 1.5)
- Copied the entirety of query and made !p test
- made interfaces.json shorter although idk if that would cause any issues

new commit goys

Added !p limbo
Fixed !p crep
Made !p crep and !p rep more user-friendly by telling the user how to use it if used incorrectly
Added reason arg for the following commands:
!p ban
!p close (Not functioning yet)
!p disband
!p kick
!p mute
!p drain
To use this, just put the reason after the command, for example: !p ban jbanate spamming
If no reason is provided, it will default to "No reason provided."
This will not show in-game (yet, idk whether to add it) and only logs in Discord

- added minMsgDelay variable and made most command delays use them
- re-added the check for a value less than 500 ms in !p crep because i managed to remove it by accident somewhehre

remove testing variable from !p cmd
!p close works now
added a 9s limit to crep duration

A PHAS PLEASE APPROV!!

### CODE REVIEW

File: src/utils/Interfaces.mjs [hunk 1]
```diff
@@ -35,14 +35,11 @@ export const MessageType = Object.freeze({
...
-  Famous: 1,
-  Splasher: 2,
-  Trusted: 2,
+  HoB, Famous: 1,
+  Splasher, Trusted: 2,
   Staff: 3,
   Admin: 4,
-  Owner: 5,
-  BotAccount: 5,
+  Owner, BotAccount: 5,
```
Comment (Conutik) [inline src/utils/Interfaces.mjs:42]: You can't do this in js, it'll give `Owner` as undefined

File: src/utils/Interfaces.mjs [hunk 2]
```diff
@@ -35,14 +35,11 @@ export const MessageType = Object.freeze({
  */
 export const Permissions = Object.freeze({
   ExSplasher: 0,
-  HoB: 1,
-  Famous: 1,
-  Splasher: 2,
-  Trusted: 2,
+  HoB, Famous: 1,
+  Splasher, Trusted: 2,
```
Comment (Conutik) [inline src/utils/Interfaces.mjs:39]: Same here

**Review by Conutik — CHANGES_REQUESTED** [review]

**Comment by jbanate** [comment]
changed also new commit:

manually select playernames.json because help me
same with generalDatabase.json and package-lock.json

Added !p limbo
Fixed !p crep
Made !p crep and !p rep more user-friendly by telling the user how to use it if used incorrectly
Added reason arg for the following commands:
!p ban
!p close (Not functioning yet)
!p disband
!p kick
!p mute
!p drain
To use this, just put the reason after the command, for example: !p ban jbanate spamming
If no reason is provided, it will default to "No reason provided."
This will not show in-game (yet, idk whether to add it) and only logs in Discord

File: src/mineflayer/commands/party/Test.mjs
Comment (Conutik) [inline src/mineflayer/commands/party/Test.mjs:1]: Why change from `misc` to `party` 😭

File: src/mineflayer/commands/party/Unban.mjs
Comment (Conutik) [inline src/mineflayer/commands/party/Unban.mjs:1]: Are you sure that the bot would be spawned in the lobby before running other commands?

File: src/mineflayer/commands/party/Ban.mjs
Comment (Conutik) [inline src/mineflayer/commands/party/Ban.mjs:1]: Same concern as unban

**Pull request merged**